guam/test/TGUAM.OPT

```diff
@@ -0,0 +1,2 @@
+  guam
+  loadhigh
```

Comment:
I'd remove this line. The sample SNAP output shows storage below the line.

Also, the test program TGUAM.MLC does not use the GUAM SVC, so "guam" is not needed.

Therefore, I'd delete guam/test/TGUAM.OPT.

See comment below for RTGUAM.BAT.